lib/govuk_sidekiq/version.rb

```diff
@@ -1,3 +1,3 @@
 module GovukSidekiq
-  VERSION = "1.0.1"
+  VERSION = "1.0.2"
```

Comment:
We do version bumps in separate PRs to avoid conflicts with other branches (https://github.com/alphagov/styleguides/blob/master/rubygems.md#versioning).